extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/ObjectStoreType.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.narayana.jta.runtime;
+
+public enum ObjectStoreType {
+    File_System,
```

Comment:
Fair enough for FS, but aren't the terms FileSystem and File System synonymous. Furthermore, isn't Camel Case more usual for config variables?

Reply:
@mmusgrov Those are enum constants, full caps is the convention in Java. So here it would be `FILE_SYSTEM`.

As I explained, that doesn't mean users will need to use that case. Quarkus converts what users give, and in this case the documentation will list possible values with the appropriate format, which in this case is hyphenated lowercase strings, so `file-system` and `jdbc`.

Reply:
ok